contrib/kube-prometheus/jsonnet/kube-prometheus/alertmanager/alertmanager.libsonnet

```diff
@@ -38,7 +47,7 @@ local k = import 'ksonnet/ksonnet.beta.3/k.libsonnet';
     secret:
       local secret = k.core.v1.secret;
 
-      secret.new('alertmanager-' + $._config.alertmanager.name, { 'alertmanager.yaml': std.base64($._config.alertmanager.config) }) +
+      secret.new('alertmanager-' + $._config.alertmanager.name, { 'alertmanager.yaml': std.base64(std.manifestYamlDoc($._config.alertmanager.config)) }) +
```

Comment:
This would be a breaking change for all the people that have overwritten their Alertmanager config.
Not really sure if we want to do this, even though I like the change personally.